libkbfs/journal_server.go

```diff
@@ -194,19 +284,39 @@ func (j *JournalServer) Enable(
 	return nil
 }
 
+// Enable turns on the write journal for the given TLF.
+func (j *JournalServer) Enable(ctx context.Context, tlfID TlfID,
+	bws TLFJournalBackgroundWorkStatus) error {
+	j.journalsLock.Lock()
+	defer j.journalsLock.Unlock()
+	return j.enableLocked(ctx, tlfID, bws, false)
+}
+
 func (j *JournalServer) dirtyOpStart(tlfID TlfID) {
-	j.lock.Lock()
-	defer j.lock.Unlock()
+	// Serialize this with other journal ops, so that
```

Comment:
I guess this works, but why not use a condvar rather than having this complicated asymmetric two locks + a channel scheme?  Seems harder to understand in my opinion, though I know you dislike condvars (not sure why).